crmd/main.c

```diff
@@ -138,6 +140,11 @@ crmd_init(void)
     state = s_crmd_fsa(C_STARTUP);
 
     if (state == S_PENDING || state == S_STARTING) {
+        if (safe_str_eq(start_state, "standby")) {
+            const crm_node_t * this_node = crm_get_peer(0, fsa_our_uname);
+            set_standby(fsa_cib_conn, this_node->uuid, NULL, "on");
```

Comment:
Scope should be XML_CIB_TAG_STATUS here, so that the option is re-evaluated at the next start.

For example, if someone sets PCMK_node_start_state="standby", starts the node (leaving it in standby), changes PCMK_node_start_state="default", then restarts pacemaker, the node will still be in standby, because the standby node attribute will still be present. Changing the scope makes the attribute go away when the cluster stops on this node.